doc-content/jbpm-docs/src/main/asciidoc/BAM/Importing-dashbuilder-data-proc.adoc

```diff
@@ -2,9 +2,9 @@
 
 ifdef::PAM,JBPM[]
 
-= Importing Dashbuilder data
-
-You can import Dashbuilder-related data to {CENTRAL} from a ZIP file if the archive is structured in the same way as the following example:
+= Importing {CENTRAL} dashboard data
+//Where does this dashboard data come from? Is it dashboard data that I created in another BC instance?
```

Comment:
Yes, it's the user created data from another BC

Reply:
Updated.